doc/ref_model/chapters/chapter05.md

```diff
@@ -226,7 +226,7 @@ This section will detail Cloud Infrastructure Software Profiles and associated c
 
 | Reference             | Feature                       | Type                       | Basic | High Performance       |
 |-----------------------|-------------------------------|----------------------------|-------|------------------------|
-| infra.net.acc.cfg.001 | vSwitch optimisation          | Yes/No and SW Optimisation | N     | Y, DPDK                |
+| infra.net.acc.cfg.001 | vSwitch optimisation (DPDK)   | Yes/No and SW Optimisation | N     | Y                      |
```

Comment:
infra.net.acc.cfg.001 The same modification must be applied in 5.13

Reply:
@karinesevilla in [5.1.3 Virtual Networking](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md#5.1.3) DPDK is listed in Description:

Reference | Feature | Type | Description | Capabilities Reference
-- | -- | -- | -- | --
infra.net.acc.cfg.001 | vSwitch optimisation | Yes/No and SW Optimisation | e.g. DPDK. | e.cap.019

Would you like that as part of the "Feature"?